handlers/officerDeltaHandler.go

```diff
@@ -1,22 +1,45 @@
 package handlers
 
 import (
+	"github.com/companieshouse/chs-delta-api/helpers"
+	"github.com/companieshouse/chs-delta-api/services"
 	"net/http"
 )
 
-// OfficerDeltaHandler offers a handler by which to publish a message onto a kafka topic.
+const (
+	officersTopic = "officers-delta"
+)
+
+// OfficerDeltaHandler offers a handler by which to publish an office-delta onto the officer-delta kafka topic.
 type OfficerDeltaHandler struct {
+	KSvc services.KafkaService
```

Comment:
We follow the standard camel case naming convention for our fields within a struct in GoLang
```suggestion
	kSvc services.KafkaService
```